docs/plugins-distribute.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Package and distribute plugins
+description: Share your custom plugins so that others can use them.
+---
+
+There is no "official" way to distribute plugins with MyST. However, you can use common workflows in the JavaScript ecosystem to distribute your plugins so that others can use them. This page documents a few approaches that may work.
```

Comment:
```suggestion
There is no "official" way to distribute plugins with MyST. However, you can use common workflows in the JavaScript ecosystem to distribute your plugins so that others can use them. This page documents a few of those.
```

Not sure if you want to be so hesitant about your recommendations :)